examples/AspNetCoreApp/Properties/launchSettings.json

```diff
@@ -17,7 +17,11 @@
       "applicationUrl": "https://localhost:7004;http://localhost:5108",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
-        "CISCO_TOKEN": "my-cisco-token"
+        "CISCO_TOKEN": "eps_Xih1gMr8qDjXOZoHZUDgo4PMzXICfUxGMcyIJ1jcfJY",
```

Comment:
is this a real token? I suggest leave here something more general to avoid exposing out inner kafka queue to the outer world